### PR TITLE
Version check for themes

### PR DESCRIPTION
**Pull request by MissAllSunday**
As per mantis report 4754:
- Checks if a mod was installed via theme install, yes, it can happen...
- Adds an < install > tag, works the same as the packager manager one.
- Uses the version tag to check for possible updates.
- Solve a couple of todo tasks.

### CODE REVIEW

File: Sources/Themes.php [hunk 1]
```diff
@@ -1366,7 +1371,7 @@ function ThemeInstall()
 		package_flush_cache();
 
 		$theme_name = $_REQUEST['copy'];
-		$images_url = $boardurl . '/Themes/' . basename($theme_dir) . '/images';
+		$images_url = $themedir . '/' . basename($theme_dir) . '/images';
```
Comment (emanuele45) [inline Sources/Themes.php:1374]: `$themedir != $boardurl . '/Themes'`

File: Sources/Themes.php [hunk 2]
```diff
@@ -1443,42 +1453,65 @@ function ThemeInstall()
...
 
-	// Something go wrong?
+	// Let us proceed with the install.
 	if ($theme_dir != '' && basename($theme_dir) != 'Themes')
 	{
 		// Defaults.
 		$install_info = array(
 			'theme_url' => $boardurl . '/Themes/' . basename($theme_dir),
-			'images_url' => isset($images_url) ? $images_url : $boardurl . '/Themes/' . basename($theme_dir) . '/images',
+			'images_url' => isset($images_url) ? $images_url : $themedir . '/' . basename($theme_dir) . '/images',
```
Comment (emanuele45) [inline Sources/Themes.php:1479]: Same as above

File: Sources/Themes.php (hunk 1 above)
Comment (MissAllSunday) [inline Sources/Themes.php:1374]: Apparently one can't make mistakes...   eww, I would offer the same "help" back but I don't give a rat ass about other projects nor do I wait to know what other people are doing so I can "borrow" it... I mind my own business... you should do the same...

Comment (emanuele45) [inline Sources/Themes.php:1374]: Fuck.
It's the first time I reach that point with a person.

I'm pointing out errors in all the code I read, if you think I didn't like you, you were wrong, now you are right.
If you prefer to have broken code in the repo, help yourself.

Comment (MissAllSunday) [inline Sources/Themes.php:1374]: Nope, you're being snarky and you're only "helping" or at least you take special consideration whenever I submit something...

I don't prefer to have broken code, thats why betas are for...

I respond the very same way I've been treated, if you keep being snarky and sarcastic this is what you're going to get, deal with it. Like I said, just mind your own business and I'll mind mine, everyones happy.

Comment (emanuele45) [inline Sources/Themes.php:1374]: OMG.
LOL

Feel free to live happily in your dreaming world.

Comment (MissAllSunday) [inline Sources/Themes.php:1374]: Now that you will refrain from having any kind of interaction with me I'll think my dreaming world will be a much better place, thanks!

**Pull request merged**